runner/runner.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function
 from azure.common.credentials import ServicePrincipalCredentials
+from azure.keyvault import KeyVaultClient
 import traceback
```

Comment:
What do you think about making this an 'as keyVault' import so all the calls go something like this keyVault.KeyVaultClient( ?

Reply:
Due to the size of the project we are unlikely to have any namespace collisions with KeyVault, so think as it is currently is fine